wip/atom-jaxb-1.0.0.buildspec

```diff
@@ -0,0 +1,26 @@
+# see instructions https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/BUILDSPEC.md
+
+# Central Repository coordinates for the Reference release
+groupId=fr.vidal.oss
+artifactId=atom-jaxb
+version=1.0.0
+
+display=${groupId}:${artifactId}
+
+# Source code
+gitRepo=git@github.com:vidal-community/atom-jaxb.git
+gitTag=v${version}
+
+# Rebuild environment prerequisites
+tool=mvn
+jdk=8
+newline=lf
+
+# Rebuild command
+command="mvn -Prelease clean install -DskipTests -Dmaven.javadoc.skip -Dgpg.skip"
```

Comment:
do not install: this is the cheat that makes you think it is reproducible :)
package or verify only

Reply:
Thank you @hboutemy, I'll have a try with `mvn package` or `mvn verify`. :hugs:

Reply:
`atom-jaxb 1.0.0` is not reproducible, as expected 🤓 (tested locally via `mvn clean verify`, see commit ae34b893ee0e8ea60ae72edd10314f347f3d4ac7):
```sh
$ ./rebuild.sh ./wip/atom-jaxb-1.0.0.buildspec

# snip ✂️ 

dos2unix: converting file target/atom-jaxb-1.0.0.buildinfo to Unix format...
dos2unix: converting file target/atom-jaxb-1.0.0.buildinfo.compare to Unix format...
rebuilding from ./wip/atom-jaxb-1.0.0.buildspec results in cat ./wip/atom-jaxb-1.0.0.buildinfo.compare:
    version=1.0.0
    ok=0
    ko=3
    okFiles=""
    koFiles="atom-jaxb-1.0.0.jar atom-jaxb-1.0.0-sources.jar atom-jaxb-1.0.0-tests.jar"
build available in ./wip/buildcache/atom-jaxb, where you can execute diffoscope
run diffoscope as container with docker run --rm -t -w /mnt -v /Users/nicolas/work/reproducible-central/wip/buildcache/atom-jaxb/./wip/atom-jaxb-1.0.0.buildspec/buildcache/atom-jaxb:/mnt:ro registry.salsa.debian.org/reproducible-builds/diffoscope
```